jsonschema-gentypes.yaml

```diff
@@ -0,0 +1,22 @@
+headers: >
+  # Automatically generated file from a JSON schema
+# Used to correctly format the generated file
+# callbacks:
+#   - - black
+#   - - isort
+generate:
+  - # JSON schema file path
+    source: src/ds_caselaw_utils/data/schema/courts.schema.json
+    # Python file path
+    destination: src/ds_caselaw_utils/courts_schema_types_autogenerated.py
+    # The name of the root element
+    root_name: RawCourtRepository
+    # Argument passed to the API
+    api_arguments:
+      additional_properties: Only explicit
+    # Rename an element
+    name_mapping: {}
+    # The minimum Python version that the code should support. By default the
+    # currently executing Python version is chosen. Note that the output
+    # may require typing_extensions to be installed.
+    python_version: "3.9"
```

Comment:
Weird that we need this in two places.

Reply:
It's probably a bug that it didn't respect this value -- I suspect this should be inverted, so that it treats the python version as the minimum of current and minimum permissable.

https://github.com/sbrunner/jsonschema-gentypes/blob/58d7b7ee112ca20fe77726b42b9594c0eb323f97/jsonschema_gentypes/__init__.py#L374

Reply:
Agree the logic is hard to follow there. Suspect you're right, but either way happy to just peg this.